src/swapnaWJun20/BankingSystem.java

```diff
@@ -0,0 +1,68 @@
+package swapnaWJun20;
+
+public class BankingSystem {
+
+	int debit, credit, balance,statement;
+	static int allDebit, allCredit, allBalance, allStatement, accHolder;
+
+	void setAmount(int amount){
+		accHolder++;
+		System.out.println(" Balance of Customer is: "+amount);
+		balance = amount;
+	}
+
+	void debitAmt(int debitAmt){
+		balance -= debitAmt;
```

Comment:
You should write the condition to verify the insufficient balance.

Reply:
Suggested changes done.